meetings/2020-11-09.md

```diff
@@ -0,0 +1,150 @@
+# PEH 2020-11-09 Meeting Agenda
+
+###### tags: `Error Handling` `Minutes`
+
+- [2020-10-26 Meeting Minutes](https://hackmd.io/@rust-libs/rJ6aitNOw)
+
+## Action Items
+
+- Implement trait based proof of concept for `Backtrace` in `core` (Jane Lusby)
+- Update `object-provider` API in generic member access RFC (Jane Lusby)
+- Move forward with adding an Error impl for `&impl Error` (Ashley Mannix)
+  - Create an FCP on the existing PR
+  - Add guidance for using &dyn Error + 'static and impl Error + Send + Sync +
+    'static for borrow and owned errors respectively
+- Review the "Fix the error trait" tracking issue to figure out next steps
+  (Oliver + Ashley Mannix)
+- Review the panic plan rfc (Jane Lusby)
+- reach out to the author of the error codes issue (Jane Lusby)
```

Comment:
@yaahc not seeing this in the past meeting notes or in this meetings minutes,  Which issue is this?

Reply:
You may find some information here: https://blog.rust-lang.org/inside-rust/2020/11/23/What-the-error-handling-project-group-is-working-on.html#one-standardized-error-trait

Reply:
@epage ping :)

Reply:
https://github.com/rust-lang/project-error-handling/issues/7

Reply:
https://github.com/rust-lang/project-error-handling/issues/7